packages/core/src/ThemeProvider/GlobalsAndReset.tsx

```diff
@@ -70,6 +70,9 @@ const GlobalsAndReset = injectSheet((theme: DefaultTheme) => ({
       cursor: "pointer",
       border: "none",
       backgroundColor: "transparent",
+      background: "inherit",
```

Comment:
I think `inherit` will be problematic. If a WP theme or parent CSS has this property, and we use just the base HTML element (which I believe happens in a couple places), that element will have whatever value the parent has.